shared_kv_storage/kv_storage/remote_lmdb/shared_kv_dbstore.py

```diff
@@ -82,7 +82,9 @@ def set(self, table, key, value):
         try:
             self._db_store.db_store_put(table, key, value)
             return True
-        except Exception:
+        except Exception as ex:
+            logger.warn("Exception occured in set - {}".
```

Comment:
is the log level function correct in case of exception?

Reply:
Do you recommend error? We ignore it anyways. I introduced the logs just for knowledge as the flow would be fine as return is taking care of it.

Reply:
s/occured/occurred/